Library/Formula/deckcontrol.rb

```diff
@@ -0,0 +1,15 @@
+require 'formula'
+
+class Deckcontrol < Formula
+    homepage 'https://github.com/bavc/deckcontrol'
```

Comment:
Double quotes everywhere, please! Run `brew audit --strict --online deckcontrol` (and likewise for the other formulae).